### PR TITLE
Rename `getWebpackOptions` to `getWebpackConfig`.

### PR DESCRIPTION
**Pull request by jaridmargolin**

### CODE REVIEW

**Comment by jaridmargolin** [comment]
One other option here could be to actually just use a getter and access via `api.webpackConfig`. Thoughts?

**Comment by jaridmargolin** [comment]
Although, `getWebpackConfig()` may better describe that work is done to get the value.

**Comment by eliperelman** [comment]
Agreed, I like having the method call over the prop.

File: docs/api/README.md [hunk 1]
```diff
@@ -289,14 +289,14 @@ api
   .then(() => console.log('all passed'));
 ```
 
-### `getWebpackOptions()`
+### `getWebpackConfig()`
 
 While tools like webpack-chain provide a convenient API for creating Webpack configurations, this is not a format that
-is understandable by Webpack. With `getWebpackOptions()`, the webpack-chain instance at `.config` will be converted to
+is understandable by Webpack. With `getWebpackConfig()`, the webpack-chain instance at `.config` will be converted to
 an configuration object readable directly by Webpack.
```
Comment (eliperelman) [inline docs/api/README.md]: Can you change "an" to "a" here?

**Pull request merged**